### PR TITLE
support register scoped style sheets for each component

### PR DESCRIPTION
**Pull request by Hanks10100**
Generate `scopeId` for each style sheet, and inject a code snippet to process them at the definition of each component.

```js
if (typeof weex === 'object' && weex && weex.document) {
  // process style sheets here
  try {
    weex.document.registerStyleSheets(scopeId, styleSheets)
  } catch (e) {}
}
```

Refer to https://github.com/Hanks10100/weex-native-directive/issues/14#issuecomment-354404107

### CODE REVIEW

File: lib/loader.js [hunk 1]
```diff
@@ -353,9 +353,11 @@ module.exports = function (content) {
   '  }\n' +
   '})\n'
 
-  // support to register static styles
-  exports += 'if (typeof __register_static_styles__ === "function") {\n' +
-  '  __register_static_styles__(__vue_options__._scopeId, __vue_styles__)\n' +
+  // support to register style sheets
+  exports += 'if (typeof weex === "object" && weex && weex.document) {\n' +
```
Comment (erha19) [inline lib/loader.js:357]: I thought the simpler judged code is `weex && weex.document` that is enough to avoid all problems.

Comment (Hanks10100) [reply lib/loader.js:357]: What if `weex` doesn't exist or equals `null` in the context? I think this validation is much safer.

Comment (erha19) [reply lib/loader.js:357]: In javascript, while `null` and `undefined` wrapped in the `if` block, they are all considered as `false`, or you can also write like this `typeof weex === "object" && weex.document`, the `weex` is unnecessary.

Comment (erha19) [reply lib/loader.js:357]: I just think this logic is a bit redundant！

**Review by erha19 — APPROVED** [review]

**Pull request merged**